sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -695,6 +695,12 @@ func withDefaultMounts(containerID string, persistentDir string) oci.SpecOpts {
 			Source:      "cgroup",
 			Options:     []string{"ro", "nosuid", "noexec", "nodev"},
 		},
+		// Bottlerocket release information for the container
+		{
+			Options:     []string{"rbind", "r"},
+			Destination: fmt.Sprintf("/etc/bottlerocket-release"),
+			Source:      fmt.Sprintf("/etc/os-release"),
+		},
```

Comment:
Is this required if it can be accessed from `/.bottlerocket/rootfs/etc/release`? Is this mount just for better developer ergonomics?

I couldn't find the `r` option [here](https://www.man7.org/linux/man-pages/man8/mount.8.html), I think the options should be:

```suggestion
		// Bottlerocket release information for the container
		{
			Destination: fmt.Sprintf("/etc/bottlerocket-release"),
			Source:      fmt.Sprintf("/etc/os-release"),
			Options:     []string{"bind", "ro"}
		},
```

`bind`: you don't need `rbind`, since nothing underneath `/etc/os-release` is mounted
`ro`: set the mount as `read-only`

Reply:
Agree with @arnaldo2792  - unless we have a need/ask for better ergonomics around this file we probably don't need to add this since it's already available! :)

Reply:
We only mount up `/.bottlerocket/rootfs/` on superpowered containers, but this file would be potentially useful on less privileged containers as well. Mounting to `/etc/bottlerocket-release` is also a bit more user friendly.

I was a little silly with the mounting options though, so I will definitely make that change!